client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -0,0 +1,397 @@
+
+import './style.scss';
+
+const DomainManagementNavigationItemContents = function( props ) {
+	const { materialIcon, text, description } = props;
+	return (
+		<React.Fragment>
+			<MaterialIcon icon={ materialIcon } className="navigation__icon" />
+			<div>
+				<div>{ text }</div>
+				<small>{ description }</small>
+			</div>
+		</React.Fragment>
+	);
+};
+
+const DomainManagementNavigationItem = function( props ) {
+	const { path, onClick, external, materialIcon, text, description } = props;
+
+	return (
+		<VerticalNavItem
+			path={ path }
+			onClick={ onClick }
+			external={ external }
+			className="navigation__nav-item"
+		>
+			<DomainManagementNavigationItemContents
+				materialIcon={ materialIcon }
+				text={ text }
+				description={ description }
+			/>
+		</VerticalNavItem>
+	);
+};
+
+class DomainManagementNavigationEnhanced extends React.Component {
+	isDomainInNormalState() {
+		const { domain } = this.props;
+		const { expired, pendingTransfer } = domain;
+		return ! pendingTransfer && ! expired;
+	}
+
+	isDomainInGracePeriod() {
+		const { domain, moment } = this.props;
+		const { expiry } = domain;
+		return moment().subtract( 18, 'days' ) <= moment( expiry );
+	}
+
+	getEmail() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ emailManagement( selectedSite.slug, domain.name ) }
+				materialIcon="email"
+				text={ translate( 'Manage your email accounts' ) }
+				description={ translate( '3 G Suite accounts' ) }
+			/>
+		);
+	}
+
+	getNameServers() {
+		const { selectedSite, translate, domain } = this.props;
+
+		if ( ! this.isDomainInNormalState() && ! this.isDomainInGracePeriod() ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementNameServers( selectedSite.slug, domain.name ) }
+				materialIcon="language"
+				text={ translate( 'Change your name servers & DNS records' ) }
+				description={ translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getDnsRecords() {
+		const { selectedSite, translate, domain } = this.props;
+
+		return (
+			<DomainManagementNavigationItem
+				path={ domainManagementDns( selectedSite.slug, domain.name ) }
+				materialIcon="language"
+				text={ translate( 'DNS records' ) }
+				description={ translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getContactsAndPrivacy() {
+		const { selectedSite, translate, domain } = this.props;
+		const { privateDomain, privacyAvailable } = domain;
+
+		if ( ! this.isDomainInNormalState() && ! this.isDomainInGracePeriod() ) {
+			return null;
+		}
+
+		let description;
+		if ( ! privacyAvailable ) {
+			description = translate( 'Privacy protection: not available' );
+		} else if ( privateDomain ) {
+			description = translate( 'Privacy protection: yes' );
+		} else {
+			description = translate( 'Privacy protection: no' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Privacy protection' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/domain-options-enhanced)